docs/src/whatsnew/latest.rst

```diff
@@ -31,6 +31,11 @@ This document explains the changes made to Iris for this release
 ✨ Features
 ===========
 
+#. `@schlunma`_ added weighted aggregation over "group coordinates":
```

Comment:
Please could you also add a `@schlunma` entry here:

https://github.com/SciTools/iris/blob/766e9e758ed8a48461deb9069e385f72f7a2c7c6/docs/src/common_links.inc#L65-L66

That will fix the documentation failures.